PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -148,6 +155,11 @@ struct HfCandidateSelectorD0 {
     // candidate DCA
     // if (candidate.chi2PCA() > cuts[pTBin][1]) return false;
 
+    // candidate topological chi2 over ndf when using KFParticle, need to add this selection to the SelectorCuts.h
```

Comment:
consider removing them if you would like to keep them commented

Reply:
The topological chi2 selection will be used in the future, but the selection needs to be optimised with MC.